### ISSUE TITLE
Missing Header and ContentInfo regions

### ISSUE THREAD

**Issue by robgasc** [ATLAS-Illinois/Illinois-Framework-Theme#1123]
**Describe the bug**
Header and footer do not have role attributes to tell assistive technology they are header and footer, respectively.

**To Reproduce**
1. Inspect the header
2. See it uses il-header element, without a role attribute
3. Inspect the footer
4. See it uses the il-footer element, without a role attribute

**Expected behavior**
il-header and il-footer should have role="header" and role="footer" applied respectively.

**Additional context**
See https://airtable.com/appRAPJCKVmGn7rwA/tblE7cPZhYPK9qvet/viwDBCVt46hi17o8t/recnYRbKJ1CMwQ0QY?blocks=hide for more details from LAS

**Comment by trubach** [comment]
This is almost certainly an issue with the Axe accessibility scanning tool. The Toolkit web components include the required region info in their shadow DOM elements, which the scanner has a hard time picking up on. I'm closing this for now.
